lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -105,6 +105,90 @@ class ConvertAtenMulOp : public OpConversionPattern<AtenMulTensorOp> {
   }
 };
 
+class ConvertAtenBatchNormOp : public OpConversionPattern<AtenBatchNormOp> {
+public:
+  using OpConversionPattern<AtenBatchNormOp>::OpConversionPattern;
+  using OpAdaptor = typename AtenBatchNormOp::Adaptor;
+
+  LogicalResult
+  matchAndRewrite(AtenBatchNormOp op, OpAdaptor adaptor,
```

Comment:
This ops takes two extra arguments, `momentum` and `training` that are currently not being handled here.

Reply:
Done, PTAL. We only support `training=false`, and ignore `momentum` and `cudnnEnabled`.
```c++
    // momentum is ignored
    Value momentum = adaptor.getMomentum();
    (void)momentum;

    // cudnnEnabled is ignored
    Value cudnnEnabled = adaptor.getCudnnEnabled();
    (void)cudnnEnabled;

    bool training = false;
    if (!matchPattern(op.getTraining(), m_TorchConstantBool(&training)))
      return rewriter.notifyMatchFailure(
        op, "non-bool training unsupported");
    if (training)
      return rewriter.notifyMatchFailure(
        op, "only inference mode batch_norm lowering supported");
```